src/NosCore.Packets/ClientPackets/GuriPacket.cs

```diff
@@ -28,17 +28,17 @@ public class GuriPacket : PacketDefinition
         [PacketIndex(0)]
         public int Type { get; set; }
 
-        [PacketIndex(1)]
+        [PacketIndex(1, IsOptional = true)]
         [UsedImplicitly]
-        public int Argument { get; set; }
+        public int? Argument { get; set; }
 
-        [PacketIndex(2)]
+        [PacketIndex(2, IsOptional = true)]
         public long? VisualEntityId { get; set; }
 
-        [PacketIndex(3)]
-        public int Data { get; set; }
+        [PacketIndex(3, IsOptional = true)]
+        public int? Data { get; set; }
 
-        [PacketIndex(4, true, false, false, SerializeToEnd = true)]
+        [PacketIndex(4, true, false, false, IsOptional = true, SerializeToEnd = true)]
         public string Value { get; set; }
     }
 }
```

Comment:
Make the 2 "int" nullable

Reply:
Isoptional on the last string is useless

Reply:
It isn't useless. If I don't put that this beautiful packet appear ->
![img](https://gyazo.com/720653c3a746adf7e12da37d520ff26d.png)
And I want perfection. The official sends "guri 2 1 1", so adding "IsOptional = true" evades this additional "-"

Reply:
It’s not suppose to show on the last packet. At least on chicken api